datasets/simulated.py

```diff
@@ -0,0 +1,87 @@
+from benchopt import BaseDataset
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy.sparse import rand as sprand
+    from scipy.signal import fftconvolve
+    from scipy.signal.windows import gaussian
+    from scipy.sparse.linalg import LinearOperator
+
+
+def make_blur(type_lin_op, size, std,
+              height):
+    if type_lin_op == 'denoising':
+        lin_op = LinearOperator(
+                dtype=np.float64,
+                matvec=lambda x: x,
+                matmat=lambda X: X,
+                rmatvec=lambda x: x,
+                rmatmat=lambda X: X,
+                shape=(height, height),
+            )
+    elif type_lin_op == 'deblurring':
+        filt = np.outer(
+            gaussian(size, std),
+            gaussian(size, std))
+        filt /= filt.sum()
+        lin_op = LinearOperator(
+                dtype=np.float64,
+                matvec=lambda x: fftconvolve(x, filt, mode='same'),
+                matmat=lambda X: fftconvolve(X, filt, mode='same'),
+                rmatvec=lambda x: fftconvolve(x, filt, mode='same'),
+                rmatmat=lambda X: fftconvolve(X, filt, mode='same'),
+                shape=(height, height),
+            )
+    return lin_op
```

Comment:
you should factorize this code in a utils file available across datasets and solvers to avoid avoid code duplication across datasets